roles/confluent.kafka_broker/templates/server.properties.j2

```diff
@@ -55,39 +55,70 @@ confluent.schema.registry.ssl.key.password={{kafka_broker_keystore_keypass}}
+
+{% if rbac_enabled|bool %}
+############################ MDS Settings #####################################
+confluent.metadata.server.listeners={{mds_http_protocol}}://0.0.0.0:{{mds_port}}
+confluent.metadata.server.advertised.listeners={{mds_http_protocol}}://{{ inventory_hostname }}:{{mds_port}}
+confluent.metadata.server.token.auth.enable=true
+confluent.metadata.server.token.max.lifetime.ms=3600000
+confluent.metadata.server.token.key.path={{rbac_enabled_private_pem_path}}
+confluent.metadata.server.public.key.path={{rbac_enabled_public_pem_path}}
+confluent.metadata.server.token.signature.algorithm=RS256
+confluent.metadata.server.authentication.method=BEARER
+{% if mds_ssl_enabled|bool %}
+confluent.metadata.server.ssl.keystore.location={{kafka_broker_keystore_path}}
+confluent.metadata.server.ssl.keystore.password={{kafka_broker_keystore_storepass}}
+confluent.metadata.server.ssl.key.password={{kafka_broker_keystore_keypass}}
+{% if mds_ssl_mutual_auth_enabled|bool %}
+confluent.metadata.server.ssl.truststore.location={{kafka_broker_truststore_path}}
+confluent.metadata.server.ssl.truststore.password={{kafka_broker_truststore_storepass}}
+confluent.metadata.server.ssl.client.auth=true
 {% endif %}
-{% if kafka_broker_listeners.external.sasl_protocol | normalize_sasl_protocol == 'GSSAPI' %}
-confluent.metrics.reporter.sasl.mechanism=GSSAPI
-confluent.metrics.reporter.sasl.kerberos.service.name={{kerberos_kafka_broker_primary}}
-confluent.metrics.reporter.sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
-   useKeyTab=true \
-   storeKey=true \
-   keyTab="{{kerberos.keytab_dir}}/{{kafka_broker_kerberos_keytab_path | basename}}" \
-   principal="{{kafka_broker_kerberos_principal}}";
 {% endif %}
-{% if kafka_broker_listeners.external.sasl_protocol | normalize_sasl_protocol == 'SCRAM-SHA-256' %}
-confluent.metrics.reporter.sasl.mechanism=SCRAM-SHA-256
-confluent.metrics.reporter.sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required \
-   username="{{sasl_scram_users.admin.principal}}" password="{{sasl_scram_users.admin.password}}";
+
+
+############################# Authorization Settings #############################
+authorizer.class.name=io.confluent.kafka.security.authorizer.ConfluentServerAuthorizer
+confluent.authorizer.access.rule.providers=CONFLUENT{% if mds_acls_enabled|bool %},ZK_ACL{% endif %}
+
+confluent.authorizer.group.provider=CONFLUENT
+super.users={{super_users | join(';')}}
+
+############################# Identity Provider Settings(LDAP - local OpenLDAP) #############################
+ldap.java.naming.factory.initial=com.sun.jndi.ldap.LdapCtxFactory
+ldap.com.sun.jndi.ldap.read.timeout=3000
+ldap.java.naming.provider.url=ldap{%if rbac_ldaps|bool%}s{% endif %}://{{rbac_ldap_url}}:{{rbac_ldap_port}}
+ldap.java.naming.security.principal=uid={{rbac_ldap_principal}},{{rbac_ldap_user_search_base}}
+ldap.java.naming.security.credentials={{rbac_ldap_princiapl_password}}
```

Comment:
variable typo, looks like its getting fixed in @JumaX  PR soon, remember to update the molecule.yml variables